.github/workflows/ci.yml

```diff
@@ -18,6 +18,21 @@ jobs:
           python3.7 -m pip install nox
       - name: Lint the code
         run: nox -s lint
+  
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python3.7 -m pip install nox
+      - name: Run the api generator
+        run: nox -s generate
```

Comment:
I think we also want to block the PR if there were changes, don't we? Can be done in a separate PR.

Reply:
> I think we also want to block the PR if there were changes, don't we

- We will add a daily cron job to update code if there are specification changes. 

- PRs shouldn't be blocked as long as the generator CI test passes, even if they include changes, daily cron job handles it.